client/components/post-list-fetcher/index.jsx

```diff
@@ -14,11 +14,12 @@ var PostListStore = require( 'lib/posts/post-list-store' ),
 
 var PostListFetcher;
 
-function dispatchQueryActions( query ) {
-	actions.queryPosts( query );
+function dispatchQueryActions( postListStoreId, query ) {
```

Comment:
Was it a conscious decision to not have `postListStoreId` be an optional second parameter, falling back to a default store ID as we did with the term actions?